editoast/src/views/timetable/stdcm.rs

```diff
@@ -238,24 +238,21 @@ async fn stdcm(
 
     // 2. Compute the earliest start time, maximum running time and maximum departure delay
     // Simulation time without stop duration
-    let (
-        simulation_run_time,
-        virtual_train_schedule,
-        virtual_train_sim_result,
-        virtual_train_pathfinding_result,
-    ) = simulate_train_run(
-        db_pool.clone(),
-        valkey_client.clone(),
-        core_client.clone(),
-        &stdcm_request,
-        &infra,
-        &rolling_stock,
-        timetable_id,
-    )
-    .await?;
-    let simulation_run_time = match simulation_run_time {
-        SimulationTimeResult::SimulationTime { value } => value,
-        SimulationTimeResult::Error { error } => {
+    let (virtual_train_schedule, virtual_train_sim_result, virtual_train_pathfinding_result) =
+        simulate_train_run(
+            db_pool.clone(),
+            valkey_client.clone(),
+            core_client.clone(),
+            &stdcm_request,
+            &infra,
+            &rolling_stock,
+            timetable_id,
+        )
+        .await?;
+
+    let simulation_run_time = match virtual_train_sim_result.clone().simulation_run_time() {
```

Comment:
I do not believe you need to `.clone()` here.

```suggestion
    let simulation_run_time = match virtual_train_sim_result.simulation_run_time() {
```

Reply:
We use it afterwards in `PathNotFoundHandler `.